### DISCUSSION TITLE
Symfony Error - Target class [Spatie\LaravelIgnition\Renderers\IgnitionExceptionRenderer] does not exist.

### DISCUSSION THREAD

**Discussion by j-fulbright** [spatie/laravel-ignition#145] [Q&A]
With the switchover to laravel-ignition, instead of getting the actual error message, I am receiving the below when we're throwing an Exception early in our code flow. In this particular case, it is due to a MySQL server issue, but I wouldnt know that since we're getting the below error.

`Target class [Spatie\LaravelIgnition\Renderers\IgnitionExceptionRenderer] does not exist.`

This is using Laravel 9 and latest laravel-ignition package. I have removed clockwork as well, just in case.

Any suggestions here, as I do not see any particular code that affects that, but maybe something is missing.

**Comment by fredrikmikkelsen** [comment]
I'm experiencing the same issue using Laravel Nova 4.0. Did you find a solution?

**Comment by bretto36** [comment]
Just making sure you've run composer install not composer install --no-dev if ignition is used as a devDependency in your composer.json

**Comment by AlaaMbark** [comment]
i face the same problem with laravel 9.19 and laravel-ignition 1.6  any help

**Comment by pricardodev** [comment]
I ran the composer update command, cleared all caches and settings and it worked again, I'm using Lavarel 11 with spatie and livewire 3

**Reply by libasseld** [reply]
You can force on composer.json  "spatie/laravel-ignition": "^2.7" deleting composer.lock and run php artisan cache:clear.

It worked for me
